lib/CGeneralTextHandler.cpp

```diff
@@ -490,4 +490,15 @@ CGeneralTextHandler::CGeneralTextHandler()
 		}
 		while (parser.endLine());
 	}
+	if (VLC->modh->modules.COMMANDERS)

```

Comment:
ZNPC00 is part of WoG, but COMMANDERS module is independent. You should handle that somehow.